etc/curl.profile

```diff
@@ -34,6 +38,7 @@ protocol inet,inet6
 seccomp
 shell none
 tracelog
+x11 none
```

Comment:
`x11 none` without `net none` (which makes no sense for curl) breaks on some systems.

```suggestion

```